infoschema/tables.go

```diff
@@ -1591,10 +1593,14 @@ var tableMemoryUsageOpsHistoryCols = []columnInfo{
 }
 
 var tableResourceGroupsCols = []columnInfo{
-	{name: "GROUP_ID", tp: mysql.TypeLonglong, size: 64, flag: mysql.NotNullFlag},
-	{name: "GROUP_NAME", tp: mysql.TypeVarchar, size: 512, flag: mysql.NotNullFlag},
-	{name: "RU_PER_SECOND", tp: mysql.TypeLonglong, size: 64},
-	// {name: "BURSTABLE", tp: mysql.TypeVarchar, size: 10},
+	{name: "NAME", tp: mysql.TypeVarchar, size: resourcegroup.MaxGroupNameLength, flag: mysql.NotNullFlag},
+	//{name: "MODE", tp: mysql.TypeVarchar, size: 12, flag: mysql.NotNullFlag},
```

Comment:
Why not clean the code if this is not used any longer?

Reply:
We might need to make them back when multiple modes are needed.

Reply:
I prefer clean the code first.

Reply:
done